src/current/cockroachcloud/export-metrics-advanced.md

```diff
@@ -0,0 +1,470 @@
+---
+title: Export Metrics From a CockroachDB Dedicated Cluster
```

Comment:
We renamed the sidebar to show "Advanced", but we still reference "Dedicated" on this page. Is this expected?

<img width="271" alt="image" src="https://github.com/cockroachdb/docs/assets/83607639/02ef49cd-5b0d-475c-ad40-7ae8737f7a7f">

Reply:
Yes that is expected. It will be fixed in [DOC-9770](https://cockroachlabs.atlassian.net/browse/DOC-9770?focusedCommentId=212855).